### PR TITLE
Queen time lock (10 hours)

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title

## Why It's Good For The Game

We have time lock for marines important role, the same should be true for the leader of the hive. You need the queen to know the game enough to be able to lead, and 10 hours seems fine to me.

Queen is even more important due to psy points, because only her can devo the crazy hivelord that is spending all the points in turrets near fob

## Changelog
:cl:
admin: Queen time role is time locked (10 hours)
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
This should be in config/policy not hardcoded

**Comment by BraveMole** [comment]
> This should be in config/policy not hardcoded

All the other time locks are hard coded

File: code/modules/mob/living/carbon/xenomorph/evolution.dm [hunk 1]
```diff
@@ -158,6 +158,11 @@
 			to_chat(src, "<span class='warning'>You are jobbanned from the Queen role.</span>")
 			return
 
+		var/datum/job/xenojob = SSjob.GetJobType(/datum/job/xenomorph/queen)
+		if(xenojob.required_playtime_remaining(client))
+			to_chat(src, "<span class='warning'>[get_exp_format(xenojob.required_playtime_remaining(client))] as [xenojob.get_exp_req_type()] required to play queen</span>")
+			return
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/evolution.dm:164]:
```suggestion
			to_chat(src, "<span class='warning'>[get_exp_format(xenojob.required_playtime_remaining(client))] as [xenojob.get_exp_req_type()] required to play the queen role.</span>")
			return
```

**Review by TiviPlus — APPROVED** [review]
This is approved by headmins

**Pull request merged**